app/services/compute_resource_key_pairs/ec2.rb

```diff
@@ -0,0 +1,4 @@
+module ComputeResourceKeyPairs
+  class EC2 < Base
```

Comment:
Not sure. Thought of creating them so we could extend the base class in the future